untitled/src/homework2/task8.java

```diff
@@ -0,0 +1,23 @@
+package homework2;
+
+import java.util.Scanner;
+
+
+public class task8 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please add positive number: ");
+        int a = sc.nextInt();
+
+        if (a <= 0) {
+            System.out.println("Number is negative or 0, try again ");
+        } else if (a > 0) {
+            int c = 0;
+            for (int i = 1; i <= a; i++) {
+                if (i % 3 == 0) {
```

Comment:
The output should contains a times numbers divisible by 3.
Also for loop count starty from 3 and update to be i+=3 (we need oonly numbers divisible by 3).

            int c = 0;
            for (int i = 3; c < a; i+=3, c++) {
                System.out.print(i);
                if( c+1 < a ) {
                    System.out.print(",");
                }
            }